runtime/src/lib.rs

```diff
@@ -863,6 +873,67 @@ impl pallet_verifiers::Config<pallet_proofofsql_verifier::ProofOfSql<Runtime>> f
     >;
 }
 
+parameter_types! {
+    pub const Coprocessor: Option<StateMachine> = Some(StateMachine::Kusama(4009));
+    pub const HostStateMachine: StateMachine = StateMachine::Substrate(*b"zkv_");
+}
+
+impl pallet_ismp::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type AdminOrigin = EnsureRoot<AccountId>;
+    type HostStateMachine = HostStateMachine;
+    type TimestampProvider = Timestamp;
+    // TODO: Update to actual currency once stable coin on relayer is established
+    type Currency = NullCurrency;
+    type Balance = Balance;
+    type Router = ModuleRouter;
+    type Coprocessor = Coprocessor;
+    type ConsensusClients = (ismp_grandpa::consensus::GrandpaConsensusClient<Runtime>,);
+    type Mmr = NoOpMmrTree<Runtime>;
+    type WeightProvider = ();
+}
+
+impl ismp_grandpa::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type IsmpHost = Ismp;
+}
+
+impl pallet_hyperbridge::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type IsmpHost = Ismp;
+}
+
+#[derive(Default)]
+pub struct ModuleRouter;
+impl IsmpRouter for ModuleRouter {
+    fn module_for_id(&self, id: Vec<u8>) -> Result<Box<dyn IsmpModule>, anyhow::Error> {
+        match id.as_slice() {
+            RECEIVING_MESSAGE_MODULE_ID => Ok(Box::new(ReceivingMessageModule::default())),
+            PALLET_HYPERBRIDGE_ID => Ok(Box::new(pallet_hyperbridge::Pallet::<Runtime>::default())),
+            _ => Err(Error::ModuleNotFound(id))?,
+        }
+    }
+}
+
+#[derive(Default)]
+struct ReceivingMessageModule;
```

Comment:
Maybe let's add a comment saying that for the moment we don't envision receiving messages but just sending them

Reply:
ok